src/modules/i18n/locales/de.json

```diff
@@ -172,7 +172,7 @@
     "drawing_attached": "Zeichnung als Anhang hinzugefügt",
     "drawing_too_large": "Ihre Zeichnung ist zu gross, entfernen Sie einige Details.",
     "drop_invalid_url": "URL ist ungültig.",
-    "drop_me_here": "KML Datei hierhin ziehen",
+    "drop_me_here": "Datei hier ablegen (KML, GPX, GeoTIFF)",
```

Comment:
There is also the placeholder to update 
![image](https://github.com/user-attachments/assets/da2023da-f830-4e84-965a-c056aaaaac91)

I've just noticed that with the KMZ support I forgot to update those translation and the placeholder, could you add as well KMZ when you already add GeoTIFF ?